src/components/PersonMetadata/PersonMetadata.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react'
+import { Alert} from 'react-bootstrap'
+import Pluralize from 'react-pluralize'
+// eslint-disable-next-line no-unused-vars
+import { PersonType } from '../PersonContainer/PersonContainer'
+import { compactNumbers } from '../../utils/helpers'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faOrcid } from '@fortawesome/free-brands-svg-icons'
+import { 
+  faQuoteLeft, 
+  faInfoCircle, 
+  faEye,
+  faDownload,
+  faScroll
+} from '@fortawesome/free-solid-svg-icons'
+import ReactHtmlParser from 'react-html-parser'
+import Link from 'next/link'
+import { orcidFromUrl } from "../../utils/helpers"
+
+
+type Props = {
+  item: PersonType
+}
+
+const PersonPresentation: React.FunctionComponent<Props> = ({item}) => {
+  if (!item ) return (
+    <Alert bsStyle="warning">
+        No content found.
+      </Alert>
+  )
+
+
+  //// Affiliation needs work in the API
+  const afilliation = () => {
+    if (item.affiliation.length < 1) { return null }
```

Comment:
This code I think should be handled in the container, make the affiliation an optional var for the presentation component.
Then you just have to check it's existance to render or not.